templates/stackage-home.hamlet

```diff
@@ -10,7 +10,7 @@ $newline never
 
     <p .stack-resolver-yaml><a href="https://docs.haskellstack.org/en/stable/GUIDE/#resolvers-and-changing-your-compiler-version">resolver</a>: #{toPathPiece name}
 
-    <h3>Hoogle
+    <p>
```

Comment:
How about removing this line entirely?

Reply:
Sure I am for that, I was just worried how it might render, but let's try it.